app/models/scopes/template_scope.rb

```diff
@@ -63,6 +63,24 @@ module TemplateScope
         unarchived.where(customization_of: nil).distinct
       end 
     }
+    # Retrieves the latest version of each customizable template
+    scope :latest_customizable, -> { 
+      family_ids = unarchived.where('org_id IN (?) OR is_default = ?', Org.where(org_type: Org.org_type_values_for(:funder).min).collect(&:id), true).pluck(:family_id)
```

Comment:
you may want to add distinct(:family_id) before.pluck otherwise we can retrieve duplicate family_id when a template is versioned... 
Also, you could reuse the already defined latest_version_per_org:
```
Template.latest_version_per_org(
Org.where(org_type: Org.where(org_type: Org.org_type_values_for(:funder).min).collect(&:id))
```
2. add a scope method for the edge case (default_template):
```
scope :latest_default, -> {
latest_version(unarchived.where(is_default: true).pluck(:family_id))
}
```
3. Chain them together where is needed